drivers/powerp-bin.c

```diff
@@ -545,7 +545,7 @@ static int powpan_updateinfo(void)
 	}
 
 	/* !OB && !TEST */
-	if (!(status.flags[0] & 0x84)) {
+	if (!(status.flags[0] & 0x84) && status.o_volt) {
```

Comment:
Note to self: wanted to ask to also bump this driver's version too, due to this change, but it does seem not-substantial as it stands now. Per PR description, effectively this removes a warning when `o_volt` is absent => is `0`... right? :)

Reply:
It's purely cosmetic. I'd get an ```output voltage too low``` on every status poll. Those calculations only make sense if `o_volt` is greater than `0`.

```
		if (status.o_volt < 0.5 * status.i_volt) {
			upsdebugx(2, "%s: output voltage too low", __func__);
		} else if (status.o_volt < 0.95 * status.i_volt) {
```

and just above it fudges the actual output voltage reported to `NUT` (depending if OL/OB).

```
		if (status.flags[0] & 0x84) {
			dstate_setinfo("output.voltage", "%s", dstate_getinfo("output.voltage.nominal"));
		} else {
			dstate_setinfo("output.voltage", "%d", op_volt(status.i_volt));
		}
```